.ci/Makefile

```diff
@@ -18,6 +21,14 @@ clean:
 env:
 	env
 
+# Create Docker image
+build-docker: validate-branch-name
```

Comment:
Is `validate-branch-name` a copy/paste leftover?

Reply:
Nope, that's good to use. It is defined below and ensures the `$(BRANCH_NAME)` var is set for this target.